Makefile

```diff
@@ -54,10 +54,6 @@ check_docker:
 check_base_files:
 	@source test/make.sh && basefiles
 
-.PHONY: check_shebangs
```

Comment:
Is there a reason we're dropping this check?

Reply:
Got it.

Reply:
For posterity, the dead code could be made to work with this change if desired:

```
diff --git a/test/make.sh b/test/make.sh
index e696ce8..e22a15e 100755
--- a/test/make.sh
+++ b/test/make.sh
@@ -18,7 +18,7 @@
 # shebang has a '- e' flag, which causes it
 # to exit on error
 function check_bash() {
-find . -name "*.sh" | while IFS= read -d '' -r file;
+find . -name "*.sh" | while IFS= read -r file;
 do
   if [[ "$file" != *"bash -e"* ]];
   then
```

The behavior is then:

```
make check_shebangs
./test/make.sh is missing shebang with -e
make: *** [check_shebangs] Error 1
```